src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,12 @@
 package mate.academy.lib;
 
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
Try to follow one code style. If in one interface you have an order Target -> Retention or vice versa, then use this code style for all custom interfaces